config/locales/server.en.yml

```diff
@@ -11,6 +11,8 @@ en:
     chat_auto_silence_from_flags_duration: "Number of minutes that users will be silenced for when they are automatically silenced due to flagged chat messages."
     chat_default_channel_id: "The chat channel that will be opened by default when a user has no unread messages or mentions in other channels."
     chat_duplicate_message_sensitivity: "The likelihood that a duplicate message by the same sender will be blocked in a short period. Decimal number between 0 and 1.0, with 1.0 being the highest setting (blocks messages more frequently in a shorter amount of time). Set to `0` to allow duplicate messages."
+    chat_minimum_message_length: "Minimum number of characters for a chat message."
+    chat_allow_attachments: "Allow attachments in public chat channels."
```

Comment:
I think we should use "uploads" instead of "attachments" for this feature, because in core "attachments" refers to non-image uploads (PDFs, DOCs, etc.).